setup.cfg

```diff
@@ -1,11 +1,42 @@
-[coverage:run]
-include = uaa_client/*
-omit = uaa_client/tests/*
+[metadata]
+name = cg-django-uaa
+version = attr: uaa_client.VERSION
+description = A cloud.gov UAA authentication backend for Django
+long_description = file: README.md, CHANGELOG.md, LICENSE.md
+long_description_content_type = md
+url = https://github.com/cloud-gov/cg-django-uaa
+license = CC0 1.0
+classifiers =   
+    Development Status :: 3 - Alpha
+    Environment :: Web Environment
+    Framework :: Django
+    Framework :: Django :: 2.2
+    Framework :: Django :: 3.0
+    Framework :: Django :: 3.1
+    Framework :: Django :: 3.2
+    Intended Audience :: Developers
+    License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Software Development :: Libraries :: Python Modules
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Topic :: Utilities
 
-[coverage:report]
-fail_under = 100
-exclude_lines = 
-    pragma: no cover
-    def __str__
-    raise NotImplementedError
-omit = uaa_client/runtests.py,uaa_client/__init__.py,uaa_client/tests/*
+[options]
+zip_safe = False
+include_package_data = True
+package_dir =
+    uaa_client = uaa_client
+packages = find:
+install_requires =
+    django>=2.2,<3.3
```

Comment:
I was leaving 2.2 because it's LTS, and pinning below 3.3 because we test up to 3.2.
We test every minor between the two, so I don't see a reason to drop any unless or until we run into problems with one